integration_tests/test_feegrant.py

```diff
@@ -153,15 +157,22 @@ def test_periodic_fee_allowance(cluster):
     )
 
     for _ in range(number_of_periods):
-        transfer(
+        tx = transfer(
             cluster,
             fee_grantee_address,
             receiver_address,
             "%s%s" % (transaction_coins, BASECRO_DENOM),
             fees="%s%s" % (fee_coins, BASECRO_DENOM),
             fee_account=fee_granter_address,
+            broadcast_mode=BLOCK_BROADCASTING,
```

Comment:
I think `block` is the default option set in `client.toml`, no?

Reply:
Yeah, one of the `client.toml` looks like this
```
chain-id = "chaintest"
keyring-backend = "test"
output = "json"
node = "tcp://localhost:10017"
broadcast-mode = "block"
```
I will remove this line then.